### DISCUSSION TITLE
File Patcher

### DISCUSSION THREAD

**Discussion by RickGre** [community/community#137847] [Other Feature Feedback, Questions, & Ideas]
It appears that there is no tech support for File Explorer? I can't seem to find any way to get help.
My version of Explorer (22621.3527.65.5) - Start Menu option - Missing the selection for Win 10 Start Menu.

**Comment by ghostinhershell** [comment]
Hi @RickGre,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on troubleshooting issues with Windows OS. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by RickGre** [comment]
Thanks

 

I did get help. New to GitHub and trying to figure out the process and where to go, but I did get a reply back.

Thank you
